thirdparty/README.md

```diff
@@ -16,6 +16,19 @@ Files extracted from upstream source:
 - `ffx_a.h` and `ffx_fsr1.h` from `ffx-fsr`
 - `license.txt`
 
+## amd-fsr2
```

Comment:
We keep two lines between each section in this file (but I've learned over time that programmers seem to be pretty oblivious of that detail :)). So add one line before, and after.

Reply:
Done.